src/screens/Customers/components/TabMenu.tsx

```diff
@@ -26,13 +26,14 @@ const styles = (theme: Theme) =>
 
 interface TabProps {
   classes: { tabs: string; tab: string; tabPanel: string };
+  defaultTab: string;
   invoices: MeterReadingRecord[];
   payments: PaymentRecord[];
 }
 
 function TabMenu(props: TabProps) {
-  const { classes, invoices, payments } = props;
-  const [value, setValue] = React.useState('0');
+  const { classes, invoices, payments, defaultTab } = props;
+  const [value, setValue] = React.useState(defaultTab);
```

Comment:
Wait is default tab not reflective enough? It's just the default number of tab this page should open on.

Reply:
Oh sorry I meant `value` and `setValue` weren't descriptive names, but I noticed that they were there before. `defaultTab` is a good name! I guess there's no need to change then if you don't want since it's out of scope of PR!